mlrun/model_monitoring/applications/context.py

```diff
@@ -86,7 +86,8 @@ def __init__(
         feature_stats: Optional[FeatureStats] = None,
     ) -> None:
         """
-        Initialize a :code:`MonitoringApplicationContext` object.
+        The :code:`_init_` function initializes a :code:`MonitoringApplicationContext` object
+        and has the following attributes.
```

Comment:
This is a method function of a class and it's `__init__`. However, this explanation adds very little information IMO

Reply:
The problem is that currently the params show but there's no context
<img width="899" alt="image" src="https://github.com/user-attachments/assets/4e1fcf8a-9d2f-4a5e-956a-c076621fbd60">
Made your fixes. Thanks!